ckcp/openshift_dev_setup.sh

```diff
@@ -76,8 +71,35 @@ parse_args() {
 }
 
 init() {
-  APP_LIST="openshift-gitops compute ckcp register-compute"
-  cluster_type="openshift"
+  APP_LIST=(
+            "openshift-gitops" 
+            "ckcp"
+           )
+  # get the list of APPS to be installed
+  read -ra APPS <<< "$(yq eval '.APPS | join(" ")' "$CONFIG")"
+  for app in  "${APPS[@]}"
+  do
+    APP_LIST+=("$app")
+  done
+
+  # get cluster type
+  cluster_type=$(yq '.CLUSTER_TYPE' "$CONFIG")
+  cluster_type=${cluster_type:-"openshift"}
+
+  GIT_URL=$(yq '.GIT_URL' "$CONFIG")
+  GIT_REF=$(yq '.GIT_REF' "$CONFIG")
+  # set default values in case they're missing
+  GIT_URL=${GIT_URL:-"https://github.com/openshift-pipelines/pipeline-service.git"}
+  GIT_REF=${GIT_REF:="main"}
+
+  # get list of CRs to sync 
+  read -ra CR_TO_SYNC <<< "$(yq eval '.CR_TO_SYNC | join(" ")' "$CONFIG")"
+  if (( "${#CR_TO_SYNC[@]}" <= 0 )); then
+    CR_TO_SYNC=(
```

Comment:
So, you're assigning some default resources to be synced to CR_TO_SYNC if that var is empty from the config file.
If so, why only deployments.apps and services? We should have Pipeline Specific resources by default as well. I think the list that you removed seems fair.

CR_TO_SYNC=(
            deployments.apps
            services
            ingresses.networking.k8s.io
            pipelines.tekton.dev
            pipelineruns.tekton.dev
            runs.tekton.dev
            tasks.tekton.dev
            networkpolicies.networking.k8s.io
          )

Reply:
The rationale was to have the basic CRs synced. And the changes proposed in this PR would make the installation of Pipeline Service components like Tekton optional. In such case where we don't want operators like Tekton, we would not want to sync the Tekton resource CRs either. 
-> This option to not have Pipeline Service deployed on the compute by modifying `config.yaml` with `ckcp` was in favour to make `ckcp` more generic.

Reply:
Right. But what I meant was if someone wanted to make ckcp generic (not for pipeline-service), they would put the required config in config.yaml. If they have not, that means, we need to offer our default service no? And our default includes Tekton components.

Reply:
Yes, makes sense. I can expand the default list then.